docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -0,0 +1,486 @@
+{
+  "$id": "https://person-api.sso.mozilla.com/profile/v2/schema",
+  "type": "object",
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "_schema": {
+      "$id": "/properties/_schema",
+      "type": "string",
+      "title": "The _schema Schema ",
+      "description": "Built-in reference to the schema URL itself. You can find the jsonschema.org validator at this URL.",
+      "default": "https://person-api.sso.mozilla.com/profile/v2/schema",
+      "examples": [
+        "https://person-api.sso.mozilla.com/profile/v2/schema"
+      ]
+    },
+    "user_id": {
+      "$id": "/properties/user_id",
+      "type": "string",
+      "title": "The User_id Schema ",
+      "description": "A guaranteed-unique and stable user_id. It is not to be interpreted or displayed to the users.",
+      "default": "",
+      "examples": [
+        "ad|Mozilla-LDAP-Dev|lmcardle"
+      ]
+    },
+    "timezone": {
+      "$id": "/properties/timezone",
+      "type": "string",
+      "title": "The Timezone Schema ",
+      "description": "The user's preferred timezone.",
+      "default": "Europe/London",
+      "examples": [
+        "Europe/London"
+      ]
+    },
+    "active": {
+      "$id": "/properties/active",
+      "type": "boolean",
+      "title": "The Active Schema ",
+      "description": "If true, the user is valid. If false, the user is considered de-activated and all access MUST be denied. Users MAY be re-activated. Profile consumers MUST check this value.",
```

Comment:
This seems like a trap, wouldn't it better to return 404 than a document saying active: false?

IMO this should be loudly highlighted in API docs and examples, I doubt we check this property :)

Reply:
I think that's a good idea. we'd still store this internally, but not require consumers to check, just return 404. It's also more in line with common practices. cc @andrewkrug

Reply:
note its a personapi thing so im keeping this in the schema